opensearch-operator/pkg/builders/dashboards.go

```diff
@@ -188,15 +188,15 @@ func NewDashboardsConfigMapForCR(cr *opsterv1.OpenSearchCluster, name string, co
 	for _, key := range keys {
 		sb.WriteString(fmt.Sprintf("%s: %s\n", key, config[key]))
 	}
-	data := sb.String()
+	datas := sb.String()
```

Comment:
```suggestion
	datas := sb.String()
```
Nitpick: data has no plural, it's always `data`, never `datas`